Source/UI/Controllers/CardViewController.swift

```diff
@@ -124,13 +129,7 @@ public class CardViewController: UIViewController,
         if suppressNextLog {
             suppressNextLog = false
         } else {
-            guard let checkoutAPIService = checkoutAPIService else {
-                return
-            }
-// Potential Task : add correlationID and logger to checkoutAPIService or Frames
-//          checkoutAPIService.logger.add(metadata: checkoutAPIService.correlationID(),
-//                                          forKey: .correlationID)
-//          checkoutAPIService.logger.log(.paymentFormPresented)
```

Comment:
no need for this line 
`//          checkoutAPIService.logger.log(.paymentFormPresented)
`

Reply:
Being marked with red, looks like this was an old comment, and its now removed, not added.

Green lines are for code added, red lines for code being removed.

Reply:
Being marked with red, looks like this was an old comment, and its now removed, not added.

Green lines are for code added, red lines for code being removed.

Reply:
Yeah I assume this and a few of the following comments were added by accident